app/src/main/java/com/khanhhuy/movie_db/adapter/CategoryAdapter.java

```diff
@@ -0,0 +1,55 @@
+package com.khanhhuy.movie_db.adapter;
+
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.khanhhuy.movie_db.BR;
+import com.khanhhuy.movie_db.R;
+import com.khanhhuy.movie_db.databinding.ItemListFragmentHomeBinding;
+import com.khanhhuy.movie_db.model.CategoryList;
+import java.util.List;
+
+/**
+ * Created by yeu_thuong on 11/5/2017.
+ */
+
+public class CategoryAdapter extends RecyclerView.Adapter<CategoryAdapter.CategoryViewHolder> {
+    private List<String> mStrings;
+
+    public CategoryAdapter(List<String> strings){
+        mStrings = strings;
+    }
+
+    @Override
+    public CategoryViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_list_fragment_home, parent, false);
+        return new CategoryViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(CategoryViewHolder holder, int position) {
+        holder.bindData(new CategoryList(mStrings.get(position)));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mStrings == null? 0 : mStrings.size();
+    }
+
+    public static class CategoryViewHolder extends RecyclerView.ViewHolder{
+        ItemListFragmentHomeBinding itemListFragmentHomeBinding;
```

Comment:
2.1 Java Style Rules

Field names need to be defined at first and follow this conventions.

Non-public, non-static field names start with m.
Static field names start with s.
Other fields start with a lower case letter.
Public static final fields (constants) are ALL_CAPS_WITH_UNDERSCORES.
Example:

public class MyClass {
    public static final int SOME_CONSTANT = 42;
    public int publicField;
    private static MyClass sSingleton;
    int mPackagePrivate;
    private int mPrivate;
    protected int mProtected;
}